added a link to Treehouse online interactive tutorial for learning co…

cheapwebmonkey opened the pull request:
added a link to Treehouse online interactive tutorial for learning code  and a link and description to my fave Haskell learning resource

cheapwebmonkey committed:
added a link to Treehouse online interactive tutorial for learning code  and a link and description to my fave Haskell learning resource.

aceyoung9 commented on learning_resources.md:
```diff
@@ -9,6 +9,11 @@
+  Learn from over 1000 videos created by expert teachers on web design, coding,
+  business, and much more. The Treehouse library is continually refreshed with
+  the latest on web technology so you'll never fall behind.
 
```
Please add a note that Treehouse is not free. I would also prefer if you wrote an original description, instead of copying marketing text. :)

aceyoung9 commented on learning_resources.md:
```diff
@@ -162,3 +167,8 @@
   take.
+
+### Functional Programming
+* [Haskell Programming from First Principles](http://haskellbook.com)
```
Please add that the first chapter is offered as a free sample, and list the price of the book.

cheapwebmonkey replied:
Added

aceyoung9 reviewed (changes requested):
We've stuck to only listing free/gratis resources so far, but I'd be willing to add these resources if you list the costs in the description. Thanks!

cheapwebmonkey committed:
added info about Treehouse not being free unless your public library participates in their program.

cheapwebmonkey committed:
added treehouse and haskell book pricing

cheapwebmonkey commented:
Added the monthly pricing for Treehouse and the info that is is free if your public library participates in their program- you can check out a license. Added the info about the Haskell book being free for the first chapter, then $59

aceyoung9 commented:
Out of curiosity, which public library has Treehouse licenses to check out? That's excellent!

cheapwebmonkey commented:
The Louisville Free Public Library offers this. It is in Louisville, KY.  I am certain other places around the country do as well. I think usually libraries that offer Lynda.com checkouts, offer Treehouse. People should check with their local branches. I had no idea our library in did until I got involved with the Code Louisville program.

aceyoung9 commented:
Wow! Thanks for sharing!
